main.tf

```diff
@@ -1,7 +1,7 @@
 module "tfc_backup_to_b2" {
   source = "github.com/silinternational/terraform-aws-tfc-backup-to-b2?ref=main"
 
-  app_env                 = data.terraform_remote_state.common.outputs.app_env
+  app_env                 = var.app_env
```

Comment:
Why this change? Wouldn't it allow for two different app_env values?

Reply:
I think that looks like a circular reference. Instead, leave line 4 of main.tf as `app_env = data.terraform_remote_state.common.outputs.app_env` but keep everything else as it is. I suppose it's all the same because I didn't realize you were using interpolation to form the "common remote" workspace name.

Reply:
I believe this should remain as it was so that the value comes from the "common" remote.